doc/module-azplugins-wall.rst

```diff
@@ -0,0 +1,25 @@
+.. Copyright (c) 2018-2020, Michael P. Howard
+.. Copyright (c) 2021-2024, Auburn University
+.. Part of azplugins, released under the BSD 3-Clause License.
+
+azplugins.wall
+--------------
+
+.. rubric:: Overview
+
+.. py:currentmodule:: hoomd.azplugins.wall
+
+.. autosummary::
+    :nosignatures:
+
+    Colloid
+    LJ93
```

Comment:
HOOMD intends to rename the LJ class more explicitly, so let's follow that convention here:
```suggestion
    LennardJones93
```